backend/FwLite/LocalWebApp/LocalWebApp.csproj

```diff
@@ -47,7 +48,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <EmbeddedResource Include="..\..\..\frontend\viewer\dist\**\*" />
+        <EmbeddedResource Include="..\..\..\frontend\viewer\dist-web-component\**\*" />
```

Comment:
Why did you change this? This project is what it used to access the app, not the component.

Reply:
Oh, I see what I did now.

The LocalWebApp wouldn't start for me, because it was complaining that it couldn't find the viewer.
I know we're using a web-component version and not the viewer app in the extension and the tasks for building the viewer in the extension task file also build the component, so it seemed clear to me that this should be targeting the web-component output. Changing this line also fixed the problem for me, because I only had the web-component built.

I guess this line is actually meaningless in the context of the extension, right?

Anyway, I'll change it back.

Reply:
That also means my changes to the taskfile are also undesireable. It's a bit unfortunate, that you have to make sure the viewer app is built, so it gets embedded, because without it the LocalWebApp won't start even though it's not really used.

At least...that's my current understanding of things 😆

Reply:
haha, yeah we could make it optional, though with my changes coming to use blazor that will probably all change anyway so lets not think too much about it lol.